railties/lib/rails/generators/generated_attribute.rb

```diff
@@ -148,6 +148,7 @@ def default
                      when :references, :belongs_to,
                           :attachment, :attachments,
                           :rich_text                   then nil
+                     when :json                        then '{"key": "value"}'
```

Comment:
Is `jsonb` included in `json`? Given that `references` and `belongs_to` are both mentioned, and `belongs_to` is an alias of `references`, my guess is that it's not.

Reply:
Unfortunately, `jsonb` is not a [valid type](https://github.com/rails/rails/blob/c3c33947160e473a3d41555f8cf3aee7f3a48c76/railties/lib/rails/generators/generated_attribute.rb#L73-L77).

Reply:
Although, now that I'm looking at that method, I'm realizing it's dynamic based on the current database adapter. I think this test suite just happens to support `json`.

Reply:
So, maybe we could support `jsonb` too.